recipe/build.sh

```diff
@@ -66,3 +66,8 @@ find . -name "*activate*.sh" -exec sed -i.bak "s|@_CONDA_PYTHON_SYSCONFIGDATA_NA
 find . -name "*activate*.sh" -exec sed -i.bak "s|@CONDA_BUILD_CROSS_COMPILATION@|${CONDA_BUILD_CROSS_COMPILATION}|g"                "{}" \;
 
 find . -name "*activate*.sh.bak" -exec rm "{}" \;
+
+# Check if (de-)activate scripts can be used in non-Bash shells (ignoring the commonly supported "local" keyword.)
+find . -name "*activate*.sh" -exec shellcheck --severity=info --format=gcc {} \; \
+    | grep -vF "In POSIX sh, 'local' is undefined. [SC" \
+    && exit 1 || true
```

Comment:
```suggestion

# Check if (de-)activate scripts can be used in non-Bash shells (ignoring the commonly supported "local" keyword.)
find . -name "*activate*.sh" -exec shellcheck -e SC3043 --severity=info --format=gcc {} \; && exit 1 || true
```

Reply:
I don't think you want that `exit 1` anymore; it was used to fail on grep returning lines.  Keeping the `|| true` would mean that shellcheck isn't really a required criterion.

Reply:
thanks fixed!